### PR TITLE
feat: Add config pages.

### LINKED ISSUE #143: Feature: Add config pages.
### Checklist

- [X] I've searched for similar issues and couldn't find anything matching
- [X] I've discussed this feature request in the [OpenIMSDK Slack](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg) and got positive feedback

### Is this feature request related to a problem?

❎ No

### Problem Description

_No response_

### Solution Description

image

### Benefits

Enhance system.

### Potential Drawbacks

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by IRONICBo**
#### 🔍 What type of PR is this?

/kind feature

#### 👀 What this PR does / why we need it:

Add config pages.

#### 🅰 Which issue(s) this PR fixes:

Fixes #143

### CODE REVIEW

File: server/internal/service/bot.go [hunk 1]
```diff
@@ -68,8 +68,8 @@ func (svc *BotService) CreateBot(cid string, params *requestparams.CreateBotPara
 		BotPort:     params.BotPort,
 		BotToken:    params.BotToken,
 		Nickname:    params.Nickname,
-		Avatar:      params.Avatar,
-		Description: params.Description,
+		Avatar:      *params.Avatar,
+		Description: *params.Description,
```
Comment (charles-chenzz) [inline server/internal/service/bot.go:72]: why only these two fields using *?

File: server/main.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ func init() {
 	db.InitMysqlDB()
 	db.InitRedisDB()
 	client.InitMinio()
-	client.InitMail()
+	// client.InitMail()
 	hooks.InitHooks()
```
Comment (charles-chenzz) [inline server/main.go:44]: if you don't need it just delete it

File: server/internal/service/bot.go (hunk 1 above)
Comment (IRONICBo) [reply server/internal/service/bot.go:72]: These param are optional, so use pointer to get empty string params.

File: server/main.go (hunk 2 above)
Comment (IRONICBo) [reply server/main.go:44]: It based on SMTP service with third-party, I annotate it to skip mail check in  openkf server start stage and easy to develop, we will enable it in release version.

**Review by cubxxw — APPROVED** [review]

**Pull request merged**